theme/layouts/_sidebar.ejs

```diff
@@ -0,0 +1,35 @@
+<%
+const query = site.queryPages;
+subPages = site.getSubPages;
+
+const navigation = [];
+for (const menuItem of site.menuItems) {
+  const page = query(({originalPathname}) => originalPathname === menuItem)[0];
+  navigation.push(subPages(page));
+}
```

Comment:
I assume here you wanted to create the full navigation tree instead of adding just 1 level, maybe:
```
const currentOriginalPath = "documentation/actions/check";
const navigation = [];
for (const menuItem of site.menuItems) {
  const page = query(({originalPathname}) => originalPathname === menuItem)[0];
  if (menuItem === currentOriginalPath.split("/")[0]) {
    page.sub_items = subPages(page);
  }
  navigation.push(page);
}
```

Reply:
Or you can filter out children of current page and reuse existing `recNav` method from `_navigation.ejs`.